.github/workflows/publish-beta-release.yml

```diff
@@ -0,0 +1,74 @@
+name: Publish Beta Release to Github Only
+
+on: 
+  workflow_dispatch:
+
+jobs:
+  save_artifacts:
+    name: 'Build VSIX Files'
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: '16.13.x'
```

Comment:
dumb q: how do we know what npm version is being used here?

Reply:
This shows up in the summary log: 
<img width="593" alt="Screen Shot 2022-08-30 at 1 33 47 PM" src="https://user-images.githubusercontent.com/9795193/187505099-1566a1c7-278c-43f2-b907-c5c439386d8b.png">

Reply:
gotcha...be aware there was some fun with having to use npm v6 when doing the vsix publishing.  Def would be cool if that problem went away and we are only on npm 8

Reply:
we use `lts/*` or `latest` so that stuff moves automatically, if you want that.  One less thing to maintain.